documentation/theme/theming-crash-course.asciidoc

```diff
@@ -42,6 +42,9 @@ Place your global styles in a separate HTML file:
 
 As shown in the previous chapter, this file should be imported to the root layout level of your application.
 
+[WARNING]
+Remember to import the `<custom-style>` from `bower_components/polymer/lib/elements/custom-style.html`. That file is always loaded when a Vaadin component is imported, but if Vaadin components are not imported, the styles might be not loaded in the browser and scope leaks could appear in polyfilled browsers.
```

Comment:
> the styles might not be loaded

The `<style>` element is always processed by the browser. The only side-effect of `<custom-style>` not being imported is that the style scoping will not work in polyfilled browsers. So the statement here is a bit misleading and vague (”might”, but not indicating when/how).